www/assets/app.css

```diff
@@ -22,8 +22,47 @@ h1, h4 {
 h1 {
   border-bottom: 2px solid #222;
   text-align: left;
+  line-height: 1.5;
 }
 
+button {
+  border-radius: 5px;
+  background: #CCC;
+  border: none;
+}
+
+p button {
+  font-size: 0.8em;
+  font-weight: bold;
+  padding: 3px 5px;
+}
+
+input,
+textarea,
+table{
+  border: 1px solid #aaa;
```

Comment:
Think this could go with https://github.com/hoodiehq/my-first-hoodie/blob/espy-styles/www/assets/app.css#L97 and avoid styling the core elements.